tests/system/action/committee/test_json_upload.py

```diff
@@ -0,0 +1,298 @@
+                            "messages": [],
+                            "data": {"name": "test", "description": "A long text"},
+                        },
+                    ],
+                },
+            },
+        )
+        assert response.json["results"][0][0]["statistics"][1] == {
+            "name": "Committees created",
+            "value": 1,
+        }
+        assert response.json["results"][0][0]["statistics"][2] == {
+            "name": "Committees updated",
+            "value": 0,
+        }
+
+    def test_json_upload_duplicate_in_db(self) -> None:
+        self.set_models({"committee/7": {"name": "test"}})
+        response = self.request("committee.json_upload", {"data": [{"name": "test"}]})
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.DONE,
+            "messages": [],
+            "data": {"name": "test", "id": 7},
+        }
+
+    def test_json_upload_date(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_name": "test meeting",
+                        "start_date": "2023-08-09",
+                        "end_date": "2023-08-10",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "test",
+                "meeting_name": "test meeting",
+                "start_date": 1691539200,
+                "end_date": 1691625600,
+            },
+        }
+
+    def test_json_upload_wrong_date(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_name": "test meeting",
+                        "start_date": "2023-08-09",
+                        "end_date": "12XX-broken",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 400)
+        assert "Could not parse 12XX-broken except date" in response.json["message"]
+
+    def test_json_upload_meeting_field_but_no_meeting_name(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_template": "testtemplate",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.ERROR,
```

Comment:
state should be WARNING and in message should be mentioned, that no meeting will be created without meeting_name.
Assert, that there will be no meeting created, but a committee will be

Reply:
I'm not in the import action here. No meeting and no committee will be created here!

Reply:
state is now WARNING. Changed the message.